{
  rethrow(t,RuntimeException.class);
  rethrow(t,Error.class);
}
